src/validation/error.messages.ts

```diff
@@ -96,6 +96,12 @@ export enum ErrorMessages {
   YEAR_LENGTH_OF_TRUST = "The year of the date the trust was created must include 4 numbers",
   DAY_LENGTH_OF_TRUST = "The day the trust was created must include 1 or 2 numbers",
   MONTH_LENGTH_OF_TRUST = "The month the trust was created must include 1 or 2 numbers",
+  DAY_OF_TRUST_CEASED = "The date the trust stopped being associated with this overseas entity must include a day",
```

Comment:
Minor, but the reading of these constant names now sounds slightly off to me.

E.g. `DAY_OF_CEASED_TRUST` sounds preferable here. What do you think?